.github/workflows/verify.yaml

```diff
@@ -0,0 +1,89 @@
+name: verify
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.21'
+
+      - name: Install ShellCheck and shfmt
+        run: |
+          sudo apt update
+          sudo apt install -y shellcheck
+          go install mvdan.cc/sh/v3/cmd/shfmt@latest
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
```

Comment:
latest is v6, please use it

Reply:
@Parthiba-Hazra GitHub automatically creates a unique token in the workflow:
https://docs.github.com/en/actions/security-guides/automatic-token-authentication